### ISSUE TITLE
NodeSchool Girls!

### ISSUE THREAD

**Issue by paulohp** [nodeschool/organizers#235]
Hello organizers!

So, I'm from Belo Horizonte, Brazil and we're trying to run a different version of NodeSchool here, called [NodeGirls](http://nodegirls.com). This is very similar to the common workshop, but we're only accepting woman signups.
The reason is simple: here we don't have a lot of woman attending on ours evenst, and NodeGirls is a try to take down that barrier.

What you guys think about?

**Comment by gyaresu** [comment]
Hi @paulohp 
I think the principle of women-only events is fantastic and I hope you have a lot of success!

I had two thoughts around problems you might (but do not necessarily) have.

 1. It's always better for a targeted group to be run by that group.
   * There should be mostly women organising and running the events
   * Do you mostly have women organising the event? 
 2. The use of the word girls can have a sexist meaning if you're aiming the event at women and not girls.
   * English language is full of misogynistic terms that we sometimes don't even recognise.
   * Much common language is hostile to women and changing that means changing our use of it.

It's important to remember that getting a more representative diversity in tech is 100% about changing ourselves and the culture, not just about the numbers.

e.g. _"What you guys think about?"_ - `guys` is a 'gendered term'. it's always better to use [gender neutral language](https://en.wikipedia.org/wiki/Wikipedia:Gender-neutral_language) like 'folks', 'noders', 'cool cats', 'enlightened colleagues' or 'people' etc.

As a white western male over 40 I recognise that culture _Works For Me™_ and it's my job to check my own privilege and behaviours. 

I hope you take my feedback not as criticism but as solidarity with your awesome intentions!

**Comment by maxogden** [comment]
@paulohp in Medellín they are also running a NodeSchool "Sólo para mujeres" ([for example](https://ti.to/medellinjs/nodeschool-for-girls)) @julianduque can you cc Naranjita? I can't find her github username sorry.

**Comment by paulohp** [comment]
Hi @gyaresu, thanks a lot for your feedback! I appreciate that, really!

1) Yes, that's the ideia. I'm just helping to run the first events. And yes, I'm organizing with a woman, @anapaulagomes.
2) Thanks a lot for your feedback on that. I'll keep my eyes open on these details.

We're are trying to change our cultures, it's not about more or less women on tech, it's about our culture and how we take cares of each others. And our objectives is bring more diversity to our community.

Thanks!

**Comment by gyaresu** [comment]
:+1:

**Comment by maxogden** [comment]
@julianduque nevermind, found @naranjitagolden on github! @naranjitagolden is the organizer of the NodeSchool sólo para mujeres in Medellín

**Comment by dominhhai** [comment]
:+1:

**Comment by mikeal** [comment]
:100:

**Comment by tzmanics** [comment]
I've been a part of a couple lady specific tech groups and I have to agree with @gyaresu that I very much prefer to stay away from using 'girl' in group names. A way to check is to flip the name to the other gender (NodeBoys) and see if that feels ok or like an adult event.

This is great though!! Thank you for putting the effort into diversifying the community!!

**Comment by anapaulagomes** [comment]
Hi, Tara.

I think the best way is asking what the ladies think about this.
As a woman, I don't care about the term "girls". But I think is important
to hear all this thoughts.
We'll ask to participants what they think about and to be sure about this
questions here.

Thank you!
Em 17/06/2015 11:37 AM, "Tara Z. Manicsic" <notifications@github.com>
escreveu:

**Comment by ruyadorno** [comment]
I can also attest that the use of the term **Girl** is mostly fine for their target demographics (Brazilians) as @anapaulagomes stepped forward to confirm.

Because this is a very localized event I wouldn't bother about changing the name but be aware that this is an issue if you guys are ever going to organize something with any level of international exposure.

All the best :smile:

**Comment by anapaulagomes** [comment]
Exactly, Ruy. But will have a meeting in the next saturday. We'll to promote a discussion about this topic. Our eyes are open about this. Thank you! :+1:

**Comment by naranjitagolden** [comment]
This topic is very deep and has a lot of approaches, if we get stuck here is a lost, because there are sociologists and anthropologists inmerse in this issue, talking and writing about women, girls, chicks, folks... Is important!, but girls works for our target,(in latin america / girl = chica = muchacha, My mom call her best friends: Chicas! muchachas!).
We run another meetup  "Women Who Code" and is full of girls (young women).

 We need to use our energy in others solutions, like comunication channels, ways of connect apprentices and follow their achievements, how they get his jobs. Less talks and more tools and code!

**Comment by iancrowther** [comment]
nodegirls.com is returning a 503 :-(
